datumaro/cli/contexts/project/__init__.py

```diff
@@ -900,6 +949,7 @@ def build_parser(parser_ctor=argparse.ArgumentParser):
     subparsers = parser.add_subparsers()
     add_subparser(subparsers, 'export', build_export_parser)
     add_subparser(subparsers, 'filter', build_filter_parser)
+    add_subparser(subparsers, 'generate', build_generate_parser)
```

Comment:
I don't think this belongs as a `project` subcommand. It has nothing to do with projects.

Reply:
Removed from `project`